src/platform/packages/shared/kbn-connector-specs/src/specs/abuseipdb.ts

```diff
@@ -38,6 +38,7 @@ export const AbuseIPDBConnector: ConnectorSpec = {
         headerField: 'Key',
       },
     },
+    'none',
```

Comment:
I am still conflicted about this @ymao1 . Does it make sense that we need to explicitly support none? Couldn't we always support this by default in addition to whatever comes from the spec?

(I added it here because I wanted to test actions for the `abuseipdb` connector.)